ckanext/privatedatasets/actions.py

```diff
@@ -114,11 +205,11 @@ def package_acquired(context, request_data):
                         context_pkg_update['user'] = user.get('name', '')
 
                         plugins.toolkit.get_action('package_update')(context_pkg_update, dataset)
-                        log.info('Allowed Users added correctly')
+                        log.info('Action -%s access to user- ended successfully' % context['method'])
```

Comment:
As a simple way of expressing what the action was instead of just saying "Action ended successfully" that, in my opinion, is too abstract or generic for a proper logging message.
Thats why i changed the original log message.